client/src/controllers/index.ts

```diff
@@ -32,6 +32,7 @@ import { TooltipController } from './TooltipController';
 import { UnsavedController } from './UnsavedController';
 import { UpgradeController } from './UpgradeController';
 import { ZoneController } from './ZoneController';
+import { LocaleController } from './LocaleController';
```

Comment:
Just a note, we should try to keep the alpha sorting for importing. I have updated this in the PR https://github.com/wagtail/wagtail/pull/12952

Not critical but just helps the code be easier to scan through/manage.

Reply:
Thank you, yeah that was unintentional, I just accepted the auto-import and missed reordering it. Is there something like isort that we can use? [Biome has it](https://biomejs.dev/analyzer/import-sorting/), but as per the discussion on Slack it's still lacking features from other tools we use.